frontend/src/components/profile/my-tickets.tsx

```diff
@@ -0,0 +1,228 @@
+          },
+        });
+      }
+    },
+  });
+
+  const updateTicketCallback = useCallback(
+    (id: string) => {
+      return () => {
+        const answers = tickets
+          .filter((item) => item.id == id)[0]
+          .item.questions.map((question) => {
+            let answer;
+            let option = null;
+            if (question.options.length > 0) {
+              option = question.options.filter(
+                (option) =>
+                  option.id == selectedProducts[id][0].answers[question.id],
+              )[0];
+              answer = option.name;
+            } else {
+              answer = selectedProducts[id][0].answers[question.id];
+            }
+
+            const data = {
+              answer: answer,
+              question: question.id,
+            };
+            if (option) {
+              data["options"] = [option.id];
+            }
+            return data;
+          });
+
+        updateTicket({
+          variables: {
+            conference: code,
+            input: {
+              id,
+              name: selectedProducts[id][0].attendeeName,
+              email: selectedProducts[id][0].attendeeEmail,
+              answers: answers,
+            },
+          },
+        });
+      };
+    },
+    [selectedProducts],
+  );
+
+  return (
+    <Box
+      sx={{
+        borderTop: "primary",
+      }}
+    >
+      <Box
+        sx={{
+          maxWidth: "container",
+          mx: "auto",
+          my: 5,
+          px: 3,
+        }}
+      >
+        <Heading mb={5} as="h2" sx={{ fontSize: 5 }}>
+          <FormattedMessage id="profile.myTickets" />
+        </Heading>
+        {tickets.length === 0 && (
+          <FormattedMessage
+            id="profile.myTickets.notickets"
+            values={{ linkTicket: <Link path="/tickets">here</Link> }}
+          />
+        )}
+        {tickets.length > 0 && (
+          <Table
+            headers={headers}
+            mobileHeaders={headers}
+            data={tickets}
+            keyGetter={(item) => item.id}
+            rowGetter={(item) => [
+              item.item.name,
+              item.name,
+              item.email,
+              <Box sx={{ m: -2 }}>
+                <Button
+                  onClick={() => setCurrentTicket({ id: item.id, show: true })}
+                  variant="small"
+                >
+                  <FormattedMessage id="profile.manageTicket" />
+                </Button>
+              </Box>,
+            ]}
+          />
+        )}
+
+        {updatedData?.updateAttendeeTicket.__typename === "AttendeeTicket" && (
+          <Alert variant="success">
+            <FormattedMessage
+              id={`profile.myTickets.update.AttendeeTicket.message`}
+            />
+          </Alert>
+        )}
+        {updatedData?.updateAttendeeTicket.__typename ===
+          "TicketReassigned" && (
+          <Alert variant="success">
+            {updatedData.updateAttendeeTicket.message}
```

Comment:
as I said above, I would do this:

```suggestion
            <FormattedMessage id="profile.tickets.ticketReassigned" defaultMessage="The ticket has been re-assigned to {email}!" />
```

so you can translate the message in italian